app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,79 @@
+package com.codesoom.assignment.application;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+
+class TaskServiceTest {
+
+    private TaskService taskService;
+    private static final String TASK_TITLE = "test";
+    private static final String UPDATE_POSTFIX = "!!!";
+
+    @BeforeEach
+    void setUp() {
+        // subject
+        taskService = new TaskService();
+
+        // fixture
+        Task task = new Task();
+        task.setTitle(TASK_TITLE);
+
+        taskService.createTask(task);
+    }
+
+    @Test
+    void getTasks() {
+        assertThat(taskService.getTasks()).hasSize(1);
+
+        Task task = taskService.getTasks().get(0);
+        assertThat(task.getTitle()).isEqualTo(TASK_TITLE);
+    }
```

Comment:
`getTasks()`가 여려번 실행되고 있는 것 같아요. 테스트하는 대상과 단정문을 분리해서 작성해보는 것은 어떨까요?
```suggestion
    @Test
    void getTasks() {
        List<Task> tasks = taskService.getTasks();

        assertThat(tasks).hasSize(1);
        assertThat(tasks.get(0).getTitle()).isEqualTo(TASK_TITLE);
    }
```